.github/CODEOWNERS

```diff
@@ -3299,19 +3274,11 @@ x-pack/solutions/observability/x-pack/solutions/observability/packages/kbn-alert
 x-pack/solutions/observability/x-pack/solutions/observability/packages/kbn-apm-types @elastic/obs-ux-infra_services-team
 x-pack/solutions/observability/x-pack/solutions/observability/packages/kbn-custom-integrations @elastic/obs-ux-logs-team
 x-pack/solutions/observability/packages/kbn-investigation-shared @elastic/obs-ux-management-team
-x-pack/solutions/observability/src/platform/packages/shared/kbn-timerange @elastic/obs-ux-logs-team
-x-pack/solutions/observability/src/platform/packages/shared/kbn-xstate-utils @elastic/obs-ux-logs-team
-x-pack/solutions/observability/packages/logs_overview @elastic/obs-ux-logs-team
 x-pack/solutions/observability/packages/observability_ai/observability_ai_common @elastic/obs-ai-assistant
 x-pack/solutions/observability/packages/observability_ai/observability_ai_server @elastic/obs-ai-assistant
-x-pack/solutions/observability/packages/observability_utils/observability_utils_browser @elastic/observability-ui
-x-pack/solutions/observability/packages/observability_utils/observability_utils_common @elastic/observability-ui
-x-pack/solutions/observability/packages/observability_utils/observability_utils_server @elastic/observability-ui
```

Comment:
Looks like these should not have been removed

Reply:
That looks correct. The packages have been moved to `x-pack/solutions/observability/packages/utils_*`.